app/tests/integration/nimbus/test_e2e_create_experiment.py

```diff
@@ -126,57 +100,25 @@ def test_create_new_experiment_remote_settings(selenium, base_url):
     live_experiments = home.tables[0]
     assert "Live" in home.active_tab_text
     for item in live_experiments.experiments:
-        if experiment_name in item.text:
+        if default_data.public_name in item.text:
             item.click()
             break
     summary_page = SummaryPage(selenium, base_url).wait_for_page_to_load()
     assert "live" in summary_page.experiment_status.lower()
 
 
-def test_create_new_experiment_remote_settings_reject(selenium, base_url):
-    experiment_name = "test_create_new_experiment_remote_settings_reject"
+def test_create_new_experiment_remote_settings_reject(selenium, base_url, default_data):
+    default_data.public_name = "test_create_new_experiment_remote_settings_reject"
```

Comment:
Oh I think my comment vanished somehow.  Are we mutating the same object that's passed to multiple tests?  Does this share data between tests?

Reply:
This should not be the case. According to the [docs](https://docs.pytest.org/en/6.2.x/fixture.html#fixture-scopes), fixtures are destroyed after each test by default.